zerver/lib/rate_limiter.py

```diff
@@ -14,23 +17,28 @@
 client = get_redis_client()
 rules = settings.RATE_LIMITING_RULES
 def _rules_for_user(user):
+    # type: (UserProfile) -> List[Tuple[int, int]]
     if user.rate_limits != "":
-        return [[int(l) for l in limit.split(':')] for limit in user.rate_limits.split(',')]
+        return [(int(limit.split(':')[0]), int(limit.split(':')[1])) for limit in user.rate_limits.split(',')] # to keep the consistency with line 18, 44 as it is immutable tuple in those cases.
```

Comment:
This line getting very long; can you clean this up to be more readable?

Also would prefer to have this change be in a different commit from the annotations.